source/MaterialXGenOgsXml/GlslFragmentGenerator.cpp

```diff
@@ -272,6 +306,8 @@ ShaderPtr GlslFragmentGenerator::generate(const string& fragmentName, ElementPtr
 
         // Emit final result
         //
+        // TODO: Emit a mayaSurfaceShaderOutput result to plug into mayaComputeSurfaceFinal
+        //
```

Comment:
Emitting a mayaSurfaceShaderOutput would allow chaining other Maya fragments that handle fog and transparency.